### ISSUE TITLE
STweep deletes global variables when used in reference assignment

### ISSUE THREAD

**Issue by tan-wei-xin-alez** [Barteling/STweep#51]
**Describe the bug**
Formatting the following code with STweep
```
// global variable list with name "TestGVL"
{attribute 'qualified_only'}
VAR_GLOBAL
	test1 : BOOL;
	test2 : BOOL;
END_VAR
```
```
FUNCTION_BLOCK TestPOU
VAR
	_ref1 : REFERENCE TO BOOL REF= TestGVL.test1;
	_ref2 : REFERENCE TO BOOL REF= TestGVL.test2;
END_VAR
```
causes the global variables in the CODESYS Declaration Editor to be automatically deleted as seen below
```
FUNCTION_BLOCK TestPOU
VAR
	_ref1 : REFERENCE TO BOOL REF= TestGVL.;
	_ref2 : REFERENCE TO BOOL REF= TestGVL.;
END_VAR
```
<br>

Even utilizing conditional formatting (doesn't matter whether via comments or pragmas) does not prevent the global variables from being deleted as seen below
```
FUNCTION_BLOCK TestPOU
VAR
	{STweep.disable}
	_ref1 : REFERENCE TO BOOL REF= TestGVL.;
	_ref2 : REFERENCE TO BOOL REF= TestGVL.;
	{STweep.enable}
END_VAR
```
<br>

Strangely enough, adding some other declaration before the reference assignment (but after the conditional formatting statement) does prevent the global variables from being deleted as seen below
```
FUNCTION_BLOCK TestPOU
VAR
 	{STweep.disable}
	test : BOOL;
	_ref1 : REFERENCE TO BOOL REF= TestGVL.test1;
	_ref2 : REFERENCE TO BOOL REF= TestGVL.test2;
	{STweep.enable}
END_VAR
```
<br>

Removing the conditional formatting statements, however, causes the global variables to be deleted as before like below
```
FUNCTION_BLOCK TestPOU
VAR
	test : BOOL;
	_ref1 : REFERENCE TO BOOL REF= TestGVL.;
	_ref2 : REFERENCE TO BOOL REF= TestGVL.;
END_VAR

```

**Expected behavior**
No deletion of the global variables regardless of whether conditional formatting statement is included as seen below
```
FUNCTION_BLOCK TestPOU
VAR
	_ref1 : REFERENCE TO BOOL REF= TestGVL.test1;
	_ref2 : REFERENCE TO BOOL REF= TestGVL.test2;
END_VAR
```

**STweep version info**
STweep formatter for Codesys 3.5.2

**IDE version info**
CODESYS V3.5 SP16 Patch 4 + (32-bit)

**STweep settings**
```
  "Settings": {
    "BlankLines": {
      "BlankLinesAfterMultiLineCase": 0,
      "BlankLinesAfterMultiLineComment": 0,
      "BlankLinesAfterMultiLineDeclaration": 0,
      "BlankLinesAfterMultiLineStatement": 1,
      "BlankLinesAfterSingleLineCase": 0,
      "BlankLinesAfterSingleLineComment": 0,
      "BlankLinesAfterSingleLineDeclaration": 0,
      "BlankLinesAfterSingleLineStatement": 0,
      "BlankLinesAfterStatementWithBody": 1,
      "BlankLinesAfterVarBlocks": 0,
      "BlankLinesAtImplementationFileEnd": 1,
      "BlankLinesBeforeMultiLineComment": 0,
      "BlankLinesBeforeMultiLineDeclaration": 0,
      "BlankLinesBeforeMultiLineStatement": 0,
      "BlankLinesBeforeSingleLineComment": 0,
      "BlankLinesBeforeSingleLineDeclaration": 0,
      "BlankLinesBeforeSingleLineStatement": 0,
      "BlankLinesBeforeStatementWithBody": 1,
      "MaximumBlankLinesToKeepInImplementationCode": 1
    },
    "LineBreaksAndWrapping": {
      "ArrayInitializerParametersLineBreakMode": 0,
      "ChainedBinaryExpressionLineBreakMode": 0,
      "ChainedMethodsLineBreakMode": 0,
      "FunctionBlockInitializerParametersLineBreakMode": 0,
      "InvocationParametersLineBreakMode": 0,
      "KeepExistingLineBreaksInImplementationCode": true,
      "LineBreakAfterLeftBracketInArrayInitializer": false,
      "LineBreakAfterLeftParenthesisInFunctionBlockInitializer": false,
      "LineBreakAfterLeftParenthesisInInvocation": false,
      "LineBreakAfterLeftParenthesisInStructureInitializer": false,
      "LineBreakAfterPragma": 1,
      "LineBreakAfterTypeKeyWord": true,
      "LineBreakBeforeLeftBracketInArrayInitializer": false,
      "LineBreakBeforeLeftParenthesisInFunctionBlockInitializer": false,
      "LineBreakBeforeLeftParenthesisInInvocation": false,
      "LineBreakBeforeLeftParenthesisInStructureInitializer": false,
      "LineBreakBeforeRightBracketInArrayInitializer": false,
      "LineBreakBeforeRightParenthesisInFunctionBlockInitializer": false,
      "LineBreakBeforeRightParenthesisInInvocation": false,
      "LineBreakBeforeRightParenthesisInStructureInitializer": false,
      "LineBreakBeforeStatementsInCase": true,
      "MaxArrayInitializersOnSingleLine": 5,
      "MaxFunctionBlockInitializerParametersOnSingleLine": 5,
      "MaxInvocationParametersOnSingleLine": 5,
      "MaxStructureInitializerParametersOnSingleLine": 5,
      "PlaceStructKeywordOnNewLine": false,
      "PreferWrapBeforeBinaryExpressionOperator": false,
      "StructureInitializerParametersLineBreakMode": 0,
      "WrapBeforeComma": false,
      "WrapLongLinesAtInImplementationCode": 150,
      "WrapLongLinesInImplementationCode": true
    },
    "Spaces": {
      "AfterComma": true,
      "AfterVariableDeclarationColon": true,
      "AroundAdditiveOperator": true,
      "AroundAssignmentOperator": true,
      "AroundAssignmentOperatorInParameterAssignInInvocationCall": false,
      "AroundAssignmentOperatorInParameterAssignInStructureAndFunctionBlockInitializers": false,
      "AroundComment": true,
      "AroundComparisonOperator": true,
      "AroundDot": false,
      "AroundEqualityOperator": true,
      "AroundTermOperator": true,
      "BeforeArrayBrackets": false,
      "BeforeArrayInitializerBrackets": false,
      "BeforeComma": false,
      "BeforeFunctionBlockInitializerParentheses": false,
      "BeforeInvocationParentheses": false,
      "BeforeSemiColon": false,
      "BeforeStructureInitializerParentheses": false,
      "BeforeSubRangeParentheses": false,
      "BeforeVariableDeclarationColon": true,
      "InsideArrayBrackets": false,
      "InsideArrayInitializerBrackets": false,
      "InsideExpressionParentheses": false,
      "InsideFunctionBlockInitializerParentheses": false,
      "InsideInvocationParentheses": false,
      "InsideStructureInitializerParentheses": false,
      "InsideSubRangeParentheses": false,
      "SpacesAroundPragma": true
    },
    "TabsIndentAlignment": {
      "AlignMultilineConstructs": {
        "ArrayInitializers": true,
        "ChainedBinaryExpression": true,
        "FunctionBlockInitializerParameterAssignments": false,
        "FunctionBlockInitializerParameters": true,
        "InvocationCallParameters": true,
        "InvocationCallVariableAssignments": false,
        "MultilineConstructsAlignmentStyleWhenTabsAreUsed": 0,
        "StructureInitializerParameterAssignments": false,
        "StructureInitializerParameters": true
      },
      "AlignSimilarCodeInColumns": {
        "AddressAssignments": true,
        "AlignSimilarCodeInColumnsAlignmentStyleWhenTabsAreUsed": 0,
        "Assignments": false,
        "EndOfLineComments": true,
        "EnumMemberInitializers": true,
        "VariableDeclaration": true,
        "VariableInitializers": true
      },
      "ContinuousLineIndentMultiplier": 1,
      "IndentCasesInCaseStatement": true,
      "IndentDerivedDataTypesInTypeDeclaration": true,
      "IndentElseCaseInCaseStatement": true,
      "IndentSize": 2,
      "IndentStatementsInCase": true,
      "IndentStyle": 0,
      "ParenthesesAlignment": {
        "ArrayInitializerParenthesesStyle": 0,
        "FunctionBlockInitializerParenthesesStyle": 0,
        "FunctionCallParenthesesStyle": 0,
        "StructureInitializerParenthesesStyle": 0
      },
      "TabSize": 2
    }
  }
```

**Comment by Barteling** [comment]
Thanks for reporting!  i'll see what's going on.

**Comment by Barteling** [comment]
I fixed this in STweep for TwinCAT 4.0.1  which is available here : www.stweep.com/releases

and I fixed this in STweep for Codesys 4.0.0 which will probably be available Monday 31 in the Codesys store.
If you are in a hurry with the STweep for Codesys Version please sent me an email www.stweep.com/contact
